fossunited/www/book-conference-ticket/index.html

```diff
@@ -0,0 +1,514 @@
+{% extends "templates/foss_base.html" %} {% block title %} {{ _("Book Conference
+Tickets") }}{% endblock %} {% block page_content %}
+<section class="ticket-form">
+  <div class="about">
+    <div class="header px-5 py-2 mt-20">
+      {% for event in events %}
+      <h3>Book Conference Ticket for {{ event.event_name }}</h3>
+      {% endfor %}
+    </div>
+    <p class="px-5 py-2">
+      Free and Open Source Software conference by the FOSS community.
+      <br />
+      <br />
+      Website: <a href="https://indiafoss.net">indiafoss.net</a>
```

Comment:
It would be good to remove this line, we will have to change it anyway as indiafoss.net will soon be archived.